x-pack/plugins/security_solution/server/lib/telemetry/sender.test.ts

```diff
@@ -85,6 +99,7 @@ describe('TelemetryEventsSender', () => {
           },
           file: {
             size: 3,
+            created: 0,
```

Comment:
Great, thanks for extending this test. I would still be keen to test for null / undefined behaviour just to make sure they don't trickle in unexpected

Reply:
Sure: https://github.com/elastic/kibana/pull/95836/commits/a00669b3ac9ae0bd0e84189a86307ac6240812b8